runtime/queries/rust/highlights.scm

```diff
@@ -210,4 +258,8 @@
 "'"
 ] @operator
 
-"?" @special
+; Not sure why I have to rewrite it here,
+; but this duplicate is needed.
+(type_identifier) @type
```

Comment:
I think this is exactly because the order was inverted

Reply:
Doesn't seem like it, I tested it on https://tree-sitter.github.io/tree-sitter/playground and it captured just fine without the duplicate. Not sure what's happening there. Are you able to replicate it locally?